shared/django_apps/core/migrations/0057_repository_created_at.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.13 on 2024-08-08 19:49
+
+from django.db import migrations
+
+"""
+BEGIN;
+--
+-- Raw SQL operation
+--
+ALTER TABLE "repos" ADD COLUMN "created_at" timestamp with time zone DEFAULT null NULL;
+COMMIT;
+"""
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0056_branch_name_trgm_idx"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            'ALTER TABLE "repos" ADD COLUMN "created_at" timestamp with time zone DEFAULT null NULL;'
```

Comment:
I don’t think you have to define any `DEFAULT` if the column is defined as `NULL`able.